pythia.sh

```diff
@@ -125,6 +125,15 @@ update_venv() {
         (>&2 echo 'Failed to run the initial "./pythia.sh deps" command.')
         exit 7
     fi
+
```

Comment:
@dumol note that the update_venv function is chevah/server specific 

We should have just a stub/noop `update_venv` function in pythhia.sh 

And inside `pythia.conf` we should define the specific code for each project.

If you don't use paver , you don't need this

Reply:
I prefer to have minimal to no changes between the repos for `pythia.sh`, this is how I've managed `brink.sh` between server and python-package repos too.

I don't mind a bit of code that's not used in this repo.